kingpin/actors/test/test_group.py

```diff
@@ -191,6 +191,13 @@ def test_build_action_group(self):
         ret = actor._build_action_group({'TEST': 'CONTEXT'})
         self.assertEquals(ret[0]._init_context, {'TEST': 'CONTEXT'})
 
+    def test_fill_contexts_in_act_invalid_context(self):
```

Comment:
Where is the actual test that nested group actors do not clobber default contexts?